pkg/tcpip/transport/raw/endpoint.go

```diff
@@ -624,6 +624,7 @@ func (e *endpoint) HandlePacket(route *stack.Route, pkt tcpip.PacketBuffer) {
 
 	networkHeader := append(buffer.View(nil), pkt.NetworkHeader...)
 	combinedVV := networkHeader.ToVectorisedView()
+	combinedVV.Append(pkt.TransportHeader.ToVectorisedView())
```

Comment:
this seems weird; the transport header is always a single view - why convert it to a vectorized view only to have the vectorized view part be immediately discarded?